docs/source/quick.rst

```diff
@@ -0,0 +1,224 @@
+###########
+Quick Start
+###########
+
+This is a quick start guide to get something up quickly to see the API in
+action.
+
+Installing
+==========
+
+We'll simply clone the git tree and use Custodia in place to quickly test it.
+Clone the repository, and run the make egg_info command to prepare the tree for
+execution::
+   $ git clone https://github.com/latchset/custodia.git
+   $ cd custodia
+   $ make egg_info
+
+Configuring
+===========
+
+We'll use a simple a bare minimum configuration to start off.
+
+Write a file named quick.conf with the followin contents (feel free to omit the
+comments)::
+
+   [global]
+   # Listen on a socket named './quick'
+   server_socket = ./quick
```

Comment:
```make run``` uses a different socket name. Can we agree on one socket name for demos and test instances? I'm a bit worried that we are confusing people otherwise.